main.tf

```diff
@@ -75,8 +77,9 @@ locals {
         tag = var.image_tag
       }
     }
+
     server = {
-      config = merge({
```

Comment:
This should still be config

https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/values.yaml#L1158